pkg/registry/generic/rest/proxy.go

```diff
@@ -54,6 +57,7 @@ func NewUpgradeAwareProxyHandler(location *url.URL, transport http.RoundTripper,
 		Transport:       transport,
 		UpgradeRequired: upgradeRequired,
 		FlushInterval:   defaultFlushInterval,
+		MaxBytesPerSec:  capabilities.Get().PerConnectionBandwidthLimitBytesPerSec,
```

Comment:
This should really be an argument to the proxy handler from the rest registry code - I don't like the generic stuff being aware of a global static capabilities object (i'm less concerned with Kube specific code being aware of that).

This is more of a utility class than a Kube specific class - anyone using this in non Kubey ways may want different behavior.  If you have a high bandwidth proxy connection that shouldn't be limited this approach forces you to bypass the default).